chassis/core/src/main/java/com/griddynamics/jagger/JaggerLauncher.java

```diff
@@ -152,32 +155,58 @@ private static void launchMaster(final URL directory) {
             @Override
             public void run() {
                 log.info("Starting Master");
-                WebApplicationContext context = loadContext(directory, MASTER_CONFIGURATION, environmentProperties);
-                Server server = new Server(getPortFrom("master.rest.http.port", 9090));
-                try {
-                    server.setHandler(getServletContextHandler(context));
-                    server.start();
-        
-                    final Coordinator coordinator = (Coordinator) context.getBean("coordinator");
-                    coordinator.waitForReady();
-                    coordinator.initialize();
-                    Master master = (Master) context.getBean("master");
-                    master.run();
-                } catch (Exception e) {
-                    log.error("Error during embedded Jetty handling.", e);
-                    throw new RuntimeException(e);
-                } finally {
+                boolean isStandByMode =
+                        Boolean.parseBoolean(environmentProperties.getProperty("realtime.enable.standby.mode"));
+                
+                if (isStandByMode) {
+                   while (isStandByMode) {
+                       environmentProperties.setProperty(TEST_CONFIG_NAME_PROP, getTestConfigName());
+                       XmlWebApplicationContext context =
+                               (XmlWebApplicationContext) loadContext(directory, MASTER_CONFIGURATION, environmentProperties);
+                       final Coordinator coordinator = (Coordinator) context.getBean("coordinator");
+                       coordinator.waitForReady();
+                       coordinator.initialize();
+                       Master master = (Master) context.getBean("master");
+                       master.run();
+                       
+                       context.destroy();
```

Comment:
Is it really? 

from the ticket: 
"
* Master doesn't start any test configuration
out of scope: Master is waiting for start command. Command will come from JaaS in the response to status request
"